src/Mod/Draft/importSVG.py

```diff
@@ -244,36 +244,46 @@ def getcolor(color):
     (r, g, b, a)
         RGBA float tuple, where each value is between 0.0 and 1.0.
     """
+    if color == "none":
+        FreeCAD.Console.PrintMessage("Color defined to 'none', defaulting to black\n")
+        return (0.0, 0.0, 0.0, 1.0)
```

Comment:
The default Alpha value in FreeCAD is zero for some reason. So in 3 places `1.0` should become `0.0`.

Reply:
Note that the color properties involved (ShapeColor, TextColor, and LineColor) actually ignore the Alpha value. So perhaps it is wiser to always return `0.0`?

Reply:
Indeed FreeCAD works with  "transparency" instead of "alpha" (inheritance from older times... several other apps and renderers work with transparency). transparency  = 1 - alpha

I would still store the transparency value here, just to unify... At some point we might use that for something

Reply:
> The default Alpha value in FreeCAD is zero for some reason. So in 3 places `1.0` should become `0.0`.

This isn't alpha so (as confirmed). ;) I changed 4 places to return 0.0, and 2 others to return 1-a.

Reply:
> I would still store the transparency value here, just to unify... At some point we might use that for something

Is that really a good idea? It results in a color property value that the user cannot set via the GUI.